packages/cli/README.md

```diff
@@ -1359,6 +1389,128 @@ OPTIONS
   --indent=indent        specify indentation for formatting JSON or YAML output
 ```
 
+<<<<<<< HEAD
+=======
```

Comment:
Left from a merge conflict?

Reply:
Yeah, I thought I had dealt with that but obviously I hadn't.

Reply:
fixed now